src/test/java/com/puppycrawl/tools/checkstyle/utils/ModuleReflectionUtilTest.java

```diff
@@ -179,6 +185,14 @@ public void testKeepEclipseHappy() {
             .isEqualTo(1);
     }
 
+    @Test
+    public void testGetCheckstyleModules() throws Exception {
+        final String filePath = getPath("InputUtil.java");
```

Comment:
```suggestion
        final String filePath = getPath("InputModuleReflectionUtilLoadInternalFilter.java");
```

Should make CI happy.